assets/runtime/env-defaults

```diff
@@ -24,6 +24,9 @@ fi
 GITLAB_SSH_HOST=${GITLAB_SSH_HOST:-$GITLAB_HOST}
 GITLAB_SSH_PORT=${GITLAB_SSH_PORT:-$GITLAB_SHELL_SSH_PORT} # for backwards compatibility
 GITLAB_SSH_PORT=${GITLAB_SSH_PORT:-22}
+SSH_HOST_KEYS_DIR=${SSH_HOST_KEYS_DIR:-${GITLAB_DATA_DIR}/ssh/}
+SSH_HOST_KEYS_DIR=${SSH_HOST_KEYS_DIR%/}/
```

Comment:
The reason I did it like that was to allow the user to define the variable with or without a trailing slash.  Then, after these statements, it would be sure to only end in one slash.